docs/source/conf.py

```diff
@@ -84,23 +86,184 @@
 
 latex_engine = "xelatex"
 
+latex_logo = "assets/Latex_Logo_FTC.png"
+
+latex_additional_files = ["assets/Latex_Footer_FTC.png", "_static/RTX.png", 'assets/FTC_Center_Stage_Title.pdf']
+
 # Disable xindy support
 # See: https://github.com/readthedocs/readthedocs.org/issues/5476
 latex_use_xindy = False
 
+gitInfo = gitinfo.get_git_info(dir="../../.")
+gitInfo = {'commit': "N/A", 'refs': 'N/A', 'author_date': 'N/A', 'author': 'N/A'} if gitInfo==None else gitInfo
+
 latex_elements = {
+    "papersize": "letterpaper",
+
```

Comment:
Are all of these blank lines in the definition necessary? There's a LOT of blank lines, and it makes reading this content really difficult. If this is not a requirement, let's collapse this variable definition.